wiki/Beatmap_Editor/Song_Setup/en.md

```diff
@@ -1,222 +1,270 @@
+Use the [timing][Timing] sections (`F6`) to change either the hit-sound set or volume individually.
 
-#### Technical
+#### Sample Set Selection
 
-At OD0, one can hit a 300 at less than 79.5ms away from exactly-on-time. On the other end of the scale, OD10 requires being less than 19.5ms away for a 300.
+Choose whether to use `Normal`, `Soft`, or `Drum` built-in sample set.
 
-Four mods can alter overall difficulty timing when activated:
+It is possible to [enable custom overrides][Custom Sample Overrides] (please put hit sounds files inside the beatmap's folder first) for custom hit sounds instead.
 
--   [Easy](/wiki/Game_Modifiers): Halves the OD value.
--   [Hard Rock](/wiki/Game_Modifiers): Multiplies the OD value by 1.4, up to a maximum of OD10.
--   [Double Time](/wiki/Game_Modifiers): The OD value doesn't change, but the 1.5x play speed causes hit windows to be 33% shorter.
--   [Half Time](/wiki/Game_Modifiers): The OD value doesn't change, but the 0.75x play speed causes hit windows to be 33% longer.
+#### Sample Set Volume
 
-**Note:** While Half Time and Double Time do not change the actual OD value, the speed's effect on hit windows will make circles seem to have a lower or higher apparent OD, respectively. In the chart below, apparent OD values are provided to allow comparison between timings with and without these mods. Note that these apparent OD values only apply to 300s on circles. Windows for 100s, 50s, and sliders (which use the 50 hit window) scale more harshly with Double Time than this apparent OD suggests, and more leniently with Half Time. Again, outside of NoMod/HR/EZ (on the left side), the OD values below are only for comparison: the actual OD value is always a number from 0 to 10.
+Set the volume of the sample set.
 
-![Detailed OD chart](img/ODTable.jpg "Detailed OD chart")
+#### Test Sample Set
 
-**Note:** The osu! timing system does not allow a [hit circle](/wiki/Hit_Objects) to be hit until the previous one has been hit or its time frame has been exceeded (resulting in a miss). With a low OD, the time frame of one circle may overlap with the next. Thus, one could hit the second note with perfect timing (after failing to hit the first note) and end up completely missing both because the time frame of the first note has not been exceeded yet.
+Click on the buttons to test out how the sample set sounds. Four buttons are provided, namely: `Normal`, `Whistle`, `Finish`, and `Clap`.
 
-### Approximate Difficulty Rating
+These sounds are additive; `Whistle` is a combination of `Normal` and `Whistle`.
 
-This is a summary of all of the settings chosen on this page. More stars mean harder maps and more [score](/wiki/Score). This is not the final star ranking of the song; it is just an approximation based on the settings you chose.
+### Misc. Toggles
 
-Audio
------
+Miscellaneous toggles.
 
-![Audio section](img/SS_Audio.jpg "Audio section")
+`Samples match playback rate (for fully-hitsounded maps)`: TODO. NO CLUE.
```

Comment:
This changes the pitch of hitsounds with DT/NC and HT
Enabling it on a keysounded map with DT/NC raises hitsound note pitch. It sounds terrible with DT and good with NC, just say that it should be unchecked.